test/blackbox-tests/test-cases/pkg/dune

```diff
@@ -12,7 +12,17 @@
 
 (cram
  (deps %{bin:git})
- (applies_to git-source opam-repository-download multiple-opam-repos))
+ (applies_to
+  git-source
```

Comment:
I'd bite the bullet and got with `applied_to :whole_subtree`, we often do that in other subsystems

Reply:
Yeah, with the rev-store using git under the hood, its fairly likely to pull in a dependency on it unknowingly.